humility-core/src/core.rs

```diff
@@ -1401,7 +1411,18 @@ impl NetCore {
             .lookup_module(*rpc_task)?
             .lookup_enum_byname(hubris, "RpcReply")?;
 
-        Ok(Self { socket, flash: HubrisFlashMap::new(hubris)? })
+        // Hard-coded socket address, based on Hubris configuration
```

Comment:
Same comment as on the Hubris side about potentially referring to oxidecomputer/oana -- and make a single constant regardless?

Reply:
Yup, done